BP/history/provinces/bp_k_dacia.txt

```diff
@@ -1,54 +1,86 @@
+# Notes:
+# ============
+# Iazyges
+# Iazyges became increasingly diffuse by the IVth century and were settled in and around the Tisza/Tisa river,
+# their tradicinal location [1,2,3]. Listed under [4] are later princes which might be accurate but I can't access
+# the text unfortunately. Also note that the Gepids are listed as only occupying the eastern bank of the Tisza river
+# in this map [5]; I suspect the "Huns" lists under Gepid control were actually Iazyges/Roxolani/Sarmatians.
+#
+# My general impression is that the Iazyges were probably absorbed into the Huns and the 
+# Amospados in WtWSMS represents the Iazyges/Roxolani/Sarmatian groups that survived the Hunnic conquests.
+#
+# [1] https://en.wikipedia.org/wiki/Iazyges#Late_history_and_legacy
+# [2] https://en.wikipedia.org/wiki/Iazyges#/media/File:Iazyges-en.png
+# [3] https://en.wikipedia.org/wiki/Iazyges#/media/File:Pannonia03_en.png
+# [4] https://en.wikipedia.org/wiki/Iazyges#List_of_princes ; E. Stein, J.-R. Palanque. "Histoire du Bas-Empire", 213
+# [5] https://upload.wikimedia.org/wikipedia/commons/0/0b/Europe_and_the_Near_East_at_476_AD.png
+# Other map (AD 125) https://upload.wikimedia.org/wikipedia/commons/b/bb/Roman_Empire_125.png showing Iazyges did not migrate from Tisa basin
+# ============
+# Heruli
+# Probably Quadi/Suebi/Sciri; definitely East German/Alan/Sarmatian mix by 476 [1].
+# Heruli probably didn't extend nearly as far as WtWSMS. [2-4] seems to more accurately represent the Heruls being bound
```

Comment:
The map from history-files seems to correspond more with the original WTWSMS setup, but still more to the south:

![image](https://user-images.githubusercontent.com/7620180/133918604-0b074adb-dc52-4e76-ad03-028c747211b8.png)

Thoughts?

Reply:
That map shows them bounded by the Carpathians. Other maps place them ore or less along the Danube around 476, not across the mountains; for example:
https://en.wikipedia.org/wiki/Heruli#/media/File:Huns450.png
https://en.wikipedia.org/wiki/Heruli#/media/File:Southeastern_Europe_in_520,_showing_the_Byzantine_Empire_under_Justin_I_and_the_Ostrogothic_kingdom.png
https://upload.wikimedia.org/wikipedia/commons/3/3e/East-Hem_477ad.jpg

While the last can be a little hard to tell because of text, the Heruls extend into the Outer Western Carpathians.

Reply:
On the Iazyges:

![Odoacer_480ad](https://user-images.githubusercontent.com/7620180/133964046-64deb1c9-c3d0-418a-b069-798cd685205b.jpg)

https://en.wikipedia.org/wiki/Odoacer#/media/File:Odoacer_480ad.jpg

Reply:
Revised version on k_dacia only:
Based on the references to 4th century graves in the Koros/Tisza region of the Transylvanian Plateau [1], Gepids should probably be based east of the Romano-Dacian settlements in former Roman Dacia [2,3], supplanting what would have been the Dacian and some of the Iazyges-speakers. Romano-Dacian provinces are Nicene, Dacian provinces are dacian_pagan, Gepid are germanic_pagan, and Iazyges are scythian_pagan.

![20210920192156_1](https://user-images.githubusercontent.com/55886300/134089697-e241d300-f92d-4f66-b3e6-552f1f276f21.jpg)
_De Facto_

![20210920192201_1](https://user-images.githubusercontent.com/55886300/134089723-c1dc291d-0343-41a3-bb75-25d62e455603.jpg)
_Culture_

![20210920192204_1](https://user-images.githubusercontent.com/55886300/134089728-acd4da4b-842b-493c-8344-f62bad1af80b.jpg)
_Religion_

[1] - https://en.wikipedia.org/wiki/Gepids#Before_the_arrival_of_the_Huns ; last paragraph
[2] - https://en.wikipedia.org/wiki/Roman_Dacia#/media/File:Roman_province_of_Dacia_(106_-_271_AD).svg
[3] - https://en.wikipedia.org/wiki/File:Romani_daci.jpg

Reply:
I like the current setup, I think the rivers makes for clean and logical borders in the region, on top of the written records.